test/smoketest_artman.py

```diff
@@ -38,58 +38,57 @@
 SUPPORTED_LANGS = ['python', 'java', 'ruby', 'nodejs', 'php', 'go', 'csharp']
```

Comment:
Can there be one place to go to that lists all the languages? This list seems to get repeated quite often.